src/Proto3/Suite/DotProto/Internal.hs

```diff
@@ -57,6 +58,9 @@ dieLines (Turtle.textToLines -> msg) = do
 -- >>> toModulePath "foo/FiLeName_underscore.and.then.some.dots.proto"
 -- Right (Path ["Foo","FiLeName_underscore","And","Then","Some","Dots"])
 --
+-- >>> toModulePath "foo/bar/././baz/../boggle.proto"
```

Comment:
I think there's still a problem if the path begins with `./` or `../`. I don't think `collapse` will do anything about those. From documentation, I think `collapse "./foo/bar" == "./foo/bar"` so you may have to simply drop `./` from the beginning if it's there.

For the case in which the path starts with `../`, you may want that to simply be an error.